tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "target": "esnext",
     "allowSyntheticDefaultImports": true,
-    "lib": ["esnext"],
+    "lib": ["esnext", "DOM"],
```

Comment:
why?

Reply:
WebAssembly types

Reply:
do we need to use typescript at all? we're not leveraging types so standard .mjs should be sufficient

Reply:
ha - i know myself well. i stripped out typescript in the network-process repo and just using vanilla js - https://github.com/ar-io/ar-io-network-process/blob/develop/test/arns.test.js

Reply:
I added it here for using the tooling in the tests, found it helpful for tests mainly - imagine it will be useful for tooling as well.

As for the network process and ant process - we never had typescript on those IIRC.

Reply:
how is it helpful in tooling?